boards/microbit_v2/bootloader/version

```diff
@@ -0,0 +1,12 @@
+tock-kernel: 96fceb1a669b7c780b3edf9853882c02e4fcdefb
+tock-bootloader: 885883c08e1099374c532d03c621e3a5a8e2fe29
```

Comment:
Commenting on the binary: we should probably be as pedantic as we can to not confuse ourselves in the future. If we change the bootloader to support this board we should change the micro number of the version too.

Reply:
Are you referring to change the version from 1.1.0 to 1.1.1 ?

Reply:
Yes.

Reply:
Done